backend/__tests__/adminGetVendorRequests.test.js

```
@@ -0,0 +1,88 @@
+    // Create a mock JWT token (this will be used to simulate authentication)
+    mockToken = jwt.sign({ userId: 'testUserId', username: 'adminUser', email: 'admin@example.com', role: 'admin' }, process.env.JWT_SECRET, { expiresIn: '1h' });
+  });
+
+  beforeEach(() => {
+    // Reset the VendorRequest.find mock before each test to ensure a clean slate
+    VendorRequest.find.mockReset();
+  });
+
+  beforeAll(() => {
+    console.error = jest.fn(); // Mock console.error
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeAll duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)